python/servo/build_commands.py

```diff
@@ -289,6 +289,10 @@ def build(self, release=False, dev=False, jobs=None, params=None,
                 print("Unsupported UWP target.")
                 sys.exit(1)
 
+            print("*** calling angle_root() ***")
+            angle_root(target_triple)
```

Comment:
The return value can be used just below this to set the LIB variable.

Reply:
Done.